sparkle/src/components/AssistantPreview2.tsx

```diff
@@ -30,96 +48,187 @@ const subtitleClassNames = {
 };
 const descriptionClassNames = {
   base: "s-font-normal s-mb-1",
-  item: "s-text-xs s-text-element-700  s-pl-1",
+  item: "s-text-xs s-text-element-700 s-pl-1 s-line-clamp-3",
   list: "s-text-base s-text-element-800",
   gallery: "s-text-base s-text-element-800",
 };
 
-export function AssistantPreview2({
-  variant,
-  title,
-  subtitle,
-  pictureUrl,
-  description,
-  actions,
-  onClick,
-}: AssistantPreviewProps) {
-  // State to manage the visibility of the play button
-  const [isHovered, setIsHovered] = useState(false);
+function renderVariantContent(
+  props: AssistantPreviewProps & { isHovered: boolean }
+) {
+  switch (props.variant) {
+    case "item":
+      return <ItemVariantContent {...props} />;
+    case "list":
+      return <ListVariantContent {...props} />;
+    case "gallery":
+      return <GalleryVariantContent {...props} />;
+    default:
+      return <></>;
+  }
+}
+
+const ItemVariantContent = (props: ItemVariantAssistantPreviewProps) => {
```

Comment:
nit: function ItemVariantContent VS const ItemVariantContent ? is there a reason to do one over the other apart from codebase consistency?